assets/js/social_course_nav.js

```diff
@@ -0,0 +1,30 @@
+(function ($, Drupal) {
+
+  Drupal.behaviors.socialCourseNav = {
+    attach: function attach(context) {
+      var toggleNav = $('.course_nav-toggle');
```

Comment:
We should limit the search scope here to prevent attaching the listener multiple times in cases of BigPipe or other AJAX changes.

```suggestion
      var toggleNav = $('.course_nav-toggle', context);
```

Reply:
Fixed.